app/definitions/rest/v1/history.ts

```diff
@@ -0,0 +1,19 @@
+import type { IMessageFromServer } from '../../IMessage';
+
+export type HistoryEndpoints = {
```

Comment:
Endpoints are `${section}.*`, so they should be on `./v1/channels.ts`, `./v1/im.ts`, `./v1/groups.ts`